### PR TITLE
Include contracts in test directory.

### PR DESCRIPTION
**Pull request by itirabasso**

### CODE REVIEW

**Comment by alcuadrado** [comment]
IMO this should be done in `buidler-truffle4` and `buidler-truffle5`. We are only adding this to ease migrations from those systems.

See: #219 and #226.

**Comment by itirabasso** [comment]
Uhm, yeah, that makes sense to me.

File: packages/buidler-truffle4/src/index.ts [hunk 1]
```diff
@@ -36,3 +42,9 @@ internalTask(TASK_TEST_SETUP_TEST_ENVIRONMENT, async (_, { pweb3 }) => {
       definition(accounts);
     });
 });
+
+internalTask(TASK_COMPILE_GET_SOURCE_PATHS, async (_, { config }) => {
+  const sources = await glob(join(config.paths.sources, "**/*.sol"));
```
Comment (alcuadrado) [inline packages/buidler-truffle4/src/index.ts]: This should be `const sources = await runSuper();`

File: packages/buidler-truffle5/src/index.ts [hunk 2]
```diff
@@ -71,3 +77,9 @@ internalTask(TASK_TEST_SETUP_TEST_ENVIRONMENT, async (_, { web3 }) => {
       definition(accounts);
     });
 });
+
+internalTask(TASK_COMPILE_GET_SOURCE_PATHS, async (_, { config }) => {
+  const sources = await glob(join(config.paths.sources, "**/*.sol"));
```
Comment (alcuadrado) [inline packages/buidler-truffle5/src/index.ts]: Same

**Pull request merged**